.gitignore

```diff
@@ -72,5 +72,6 @@ coverage.xml
 # Front-End #
 #############
 node_modules/
+lib/
```

Comment:
Seems like most people add an `.npmignore` which doesn't feel great because it's more clutter in the repo but it's better than the alternative I guess.